plugins/obs-ffmpeg/obs-ffmpeg-srt.h

```diff
@@ -801,8 +801,8 @@ static int libsrt_write(URLContext *h, const uint8_t *buf, int size)
 #endif
 		if (time > (s->time + 60.0)) {
 			srt_bistats(s->fd, &perf, 0, 1);
-			blog(LOG_INFO,
-			     "[obs-ffmpeg mpegts muxer / libsrt] : rtt [%.2f ms], link bandwidth [%.1f Mbps]\n",
+			blog(LOG_DEBUG,
```

Comment:
i'd prefer to keep this one to info rather than debug, that's useful health info during the stream. It logs currently every minute. You find this too spammy ?

Reply:
I changed it to debug, as users can be directed to launch OBS with --verbose in order to log this message. If it were to only log when some network issue detection logic triggers, I would say it's okay for it to be INFO. But there are people who run OBS streaming 24/7, and any message logged on an interval will eventually create an incredibly large log file.

Reply:
ok works for me then